### ISSUE TITLE
[Android] Broken-looking context menus.

### ISSUE THREAD

**Issue by zingmars** [zulip/zulip-mobile#2365]
Context menus in some places seem to be very broken looking.

![screenshot-menu-1](https://user-images.githubusercontent.com/917126/39050687-ea4669ba-44ae-11e8-9003-ee612f5cbbdd.jpg)
![screenshot-menu-2](https://user-images.githubusercontent.com/917126/39050694-edf38976-44ae-11e8-8531-a34308d3148b.jpg)

This is on Android 8.0, OnePlus A3003. App version v12.3.83 from GitHub Releases.

**Comment by borisyankov** [comment]
Thanks for reporting this. I thought we fixed that, I guess not.

**Comment by pulkonet** [comment]
@borisyankov This would only be fixed via expo and couldn't be handled on our part. Expo's component do not support OnePlus Slate font very well and that's the only reason why this and other expo components used look broken on OnePlus and Oppo phones. 

A little suggestion @zingmars, if you wanna overcome this -  Change your font to Roboto (it is available in settings>font). That is the only fix until expo updates their components and provide support for OnePlus slate font.

**Comment by zingmars** [comment]
Yup, the menus show up fine with Roboto. Is there an issue about this in expo's vcs that I can track?

**Comment by pulkonet** [comment]
@zingmars [Here](https://github.com/expo/expo/issues/1186) it is. It is not for action sheet but similar as this issue exists in all of the expo components.

**Comment by jainkuniya** [comment]
@zulipbot claim

**Comment by gnprice** [comment]
The root issue here appears to be facebook/react-native#15114 .

Still not clear to me if this is the fault of RN, or of those fonts, or what.

**Comment by gnprice** [comment]
Small update FFR, since we just ran into the same underlying upstream bug in #3533:

It looks like the key thing that triggers the bug is **setting `fontWeight`**. E.g., trying to make text bold.

In particular:
* Some recent debugging [on the upstream bug](https://github.com/facebook/react-native/issues/15114#issuecomment-495318368) finds that that's the key variable.
* We worked around this issue originally by updating `react-native-action-sheet` to v1.1.0... and one of the few things that changed in that version was that it stopped setting `fontWeight`: https://github.com/expo/react-native-action-sheet/pull/52/files So that fits.
